redaxo/src/addons/be_style/boot.php

```diff
@@ -21,21 +21,19 @@
         }
     });
 
+    rex_extension::register('BE_STYLE_SCSS_COMPILE', function (rex_extension_point $ep) {
+        $subject = $ep->getSubject();
+        $subject[] = [
+            'scss_files' => rex_extension::registerPoint(new rex_extension_point('BE_STYLE_SCSS_FILES', [$this->getPath('scss/master.scss')])),
```

Comment:
Nochmal kurz für mich: wann muss ich mich an welchen der beiden EPs drann hängen?

Reply:
`BE_STYLE_SCSS_FILES` wenn du, wie bisher auch möglich war, scss dateien zum Compile-Vorgang der styles.css von be_style hinzufügen möchtest.

`BE_STYLE_SCSS_COMPILE` wenn du eine komplett eigene css datei kompilieren möchtest. Beispielsweise könnten sich Addons hier einhängen um für sich eine eigene CSS datei basierend auf deren scss zu kompilieren.

Reply:
sollte man am besten zum release mit in die docs aufnehmen.

Reply:
Wenn man mit in die Haupt-CSS compiliert, dann sind die Änderungen weg, sobald be_style aktualisiert wird, da es dann ja wieder eine neue css mitliefert.
Vorteil von der Variante ist aber, dass man einfach SCSS-Variablen der Haupt-CSS ändern kann.
(@tbaddade bitte korrigieren, falls ich was falsches schreibe)

Reply:
Diese unterschiede und use-cases sollten wir im addon dokumentieren... readme?

Reply:
Ich würde diese lieber in die Doku (https://redaxo.org/doku/master/extension-points) aufnehmen. Ich glaube ein Anwender (auch ich) schaut eher in die Auflistung der Doku, als sich durch die readme dateien oder source code der addons "durchzuklicken".

Reply:
Würde beides bevorzugen, doku und readme. Die Leute benutzen REDAXO so unterschiedlich und Hauptsache, man stolpert über die Info.